src/ostorlab/runtimes/lite_local/runtime.py

```diff
@@ -0,0 +1,454 @@
+        """Start the tracker agent to handle the scan lifecycle."""
+        tracker_agent_settings = definitions.AgentSettings(key=TRACKER_AGENT_DEFAULT)
+        self._start_agent(agent=tracker_agent_settings, extra_configs=[])
+
+    def _start_persist_vulnz_agent(self):
+        """Start the local persistence agent to dump vulnerabilities in the local config."""
+        persist_vulnz_agent_settings = definitions.AgentSettings(
+            key=LOCAL_PERSIST_VULNZ_AGENT_DEFAULT,
+            mounts=[])
+        self._start_agent(agent=persist_vulnz_agent_settings, extra_configs=[])
+
+    def _inject_asset(self, asset: base_asset.Asset):
+        """Injects the scan target assets."""
+        asset_config = self._docker_client.configs.create(name=f'asset_{self.name}',
+                                                          labels={'ostorlab.universe': self.name},
+                                                          data=asset.to_proto())
+        asset_config_reference = docker.types.ConfigReference(config_id=asset_config.id,
+                                                              config_name=f'asset_{self.name}',
+                                                              filename='/tmp/asset.binproto')
+        selector_config = self._docker_client.configs.create(name=f'asset_selector_{self.name}',
+                                                             labels={'ostorlab.universe': self.name},
+                                                             data=asset.selector)
+        selector_config_reference = docker.types.ConfigReference(config_id=selector_config.id,
+                                                                 config_name=f'asset_selector_{self.name}',
+                                                                 filename='/tmp/asset_selector.txt')
+        inject_asset_agent_settings = definitions.AgentSettings(key=ASSET_INJECTION_AGENT_DEFAULT,
+                                                                restart_policy='none')
+        self._start_agent(agent=inject_asset_agent_settings,
+                          extra_configs=[asset_config_reference, selector_config_reference])
+
+    def _scale_service(self, service: docker_models_services.Service, replicas: int) -> None:
+        """Calling scale directly on the service causes an API error. This is a workaround that simulates refreshing
+         the service object, then calling the scale API."""
+        for s in self._docker_client.services.list():
+            if s.name == service.name:
+                s.scale(replicas)
+
+    def list(self, page: int = 1, number_elements: int = 10) -> List[runtime.Scan]:
+        """Lists scans managed by runtime.
+
+        Args:
+            page: Page number for list pagination (default 1).
+            number_elements: count of elements to show in the listed page (default 10).
+
+        Returns:
+            List of scan objects.
+        """
+        if page is not None:
```

Comment:
This is method is not supported in the lite local runtime. It should throw a NotImplementedError exception.